### PR TITLE
Changing board names to prevent confusion

### PR DESCRIPTION
**Pull request by blindassassin111**
## Description

Changed names of 3 boards I designed to remove the reference to a name that will cause confusion in the future as I want to have a product use that name but given the conflict, I decided to rename these to PCB_BH to help with possible confusion.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/omnikey_bh/readme.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
-Omnikey Blackheart PCB
+Omnikey BH PCB
 ===
 
 A replacement PCB for Omnikey keyboards. Supports 101, 102, Plus, Ultra T, Ultra, Prime and Stellar, as well as customs.
 
 Keyboard Maintainer: QMK Community and blindassassin111  
-Hardware Supported: Omnikey blackheart PCB  
+Hardware Supported: Omnikey bh PCB
```
Comment (noroadsleft) [inline keyboards/omnikey_bh/readme.md]:
```suggestion
Hardware Supported: Omnikey BH PCB  
```

File: keyboards/z150_bh/readme.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
-Z-150 Blackheart PCB
+Z-150 BH PCB
 ===
 
 A replacement PCB for Zenith Z-150 keyboards.
 
 Keyboard Maintainer: QMK Community and blindassassin111  
-Hardware Supported: Z-150 blackheart PCB  
+Hardware Supported: Z-150 bh PCB
```
Comment (noroadsleft) [inline keyboards/z150_bh/readme.md]:
```suggestion
Hardware Supported: Z-150 BH PCB  
```

**Comment by noroadsleft** [comment]
I've labeled this `breaking_change` as this changes the firmware build instructions for anyone who already has one of these PCBs. This change needs to be conveyed to the already-established end users.

**Comment by skullydazed** [comment]
Since you're changing these anyway, and you have at least one more PCB on the way, would it make sense to group these under one directory? One of our long term goals is to get everything organized as `keyboards/<manufacturer>/<keyboard_name>` and this would save us some work later. :)

**Comment by blindassassin111** [comment]
> Since you're changing these anyway, and you have at least one more PCB on the way, would it make sense to group these under one directory? One of our long term goals is to get everything organized as `keyboards/<manufacturer>/<keyboard_name>` and this would save us some work later. :)

I don't think it would make sense for this specific scenario, as I don't plan on doing another pcb like this I just regret erroneously adding the blackheart name to the end of the pcbs as I want to use that name for an actual project rather than tagging it on these just because I didn't want to just label them what the original manus did.

**Comment by blindassassin111** [comment]
Is there anything I need to do in order to get this pushed through? I made the changes requested yet nothing so unsure of where to go from here.

File: keyboards/at101_bh/at101_bh.h [hunk 3]
```diff
@@ -1,5 +1,5 @@
-#ifndef at101_blackheart_H
-#define at101_blackheart_H
+#ifndef at101_bh_H
+#define at101_bh_H
```
Comment (fauxpark) [inline keyboards/at101_bh/at101_bh.h:2]:
```suggestion
#pragma once
```

File: keyboards/omnikey_bh/omnikey_bh.h [hunk 4]
```diff
@@ -1,5 +1,5 @@
-#ifndef omnikey_blackheart
-#define omnikey_blackheart
+#ifndef omnikey_bh
+#define omnikey_bh
```
Comment (fauxpark) [inline keyboards/omnikey_bh/omnikey_bh.h:2]:
```suggestion
#pragma once
```

**Comment by noroadsleft** [comment]
I've retargeted this to the `future` branch so as to use that workflow to put a changelog entry on this, but I may revisit that decision in the next couple of days.

**Comment by noroadsleft** [comment]
Going to go ahead and merge this. This Breaking Changes cycle has a couple of similar PRs in it, and I'll write a changelog for all of them together.

Thanks!

**Pull request merged**